flow360/plugins/report/report_items.py

```diff
@@ -1211,6 +1229,34 @@ def _get_figures(self, cases, context: ReportContext):
 
         return file_names, data.x_label, data.y_label
 
+    def _check_2d_caption_validity(self, cases):
+        """Checks the validity of caption input for a given Chart2D instance."""
+
+        if isinstance(self.caption, PatternCaption):
+            raise ValueError("PatternCaption is not supported for Chart2D.")
```

Comment:
If that is the case, modify Chart2D and Chart3D models to accept different types